README.md

```diff
@@ -137,6 +139,12 @@ The tool (codecrumbs) allows us to learn, document and explain a codebase much f
 
 The ultimate goal is to have many case studies hosting at [https://codecrumbs.io](https://codecrumbs.io/). **The library of projects "explained with codecrumbs", the place for collaborative learning**. More features around that coming soon, stay tuned.
 
+## Docker
```

Comment:
maybe it's better to put into "Get started" section. So basically you have 2 ways:
- if you have node etc. (you'r JS dev) then just go with `yarn add ...`
- you don't have anything and don't wanna bother - here is Docker for you
What do you think?